README.md

```diff
@@ -45,11 +45,11 @@ This provider's versions are compatible with the following versions of Cluster A
 
 This provider's versions are able to install and manage the following versions of Kubernetes:
 
-| | Kubernetes 1.13 | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 | Kubernetes 1.19 | Kubernetes 1.20 |
-|-------------------------------------------------|---|---|---|---|---|---|---|---|
-|OpenStack Provider v1alpha1 (release-0.1 branch) | ✓ | ✓ | ✓ |   |   |   |   |   |
-|OpenStack Provider v1alpha2 (v0.2)               |   |   | ✓ |   |   |   |   |   |
-|OpenStack Provider v1alpha3 (v0.3)               |   |   |   | ✓ | ✓ | ✓ | ✓ | ✓ |
+| | Kubernetes 1.13 | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 | Kubernetes 1.19 | Kubernetes 1.20 | Kubernetes 1.21 |
+|-------------------------------------------------|---|---|---|---|---|---|---|---|---|
+|OpenStack Provider v1alpha1 (release-0.1 branch) | ✓ | ✓ | ✓ |   |   |   |   |   |   |
+|OpenStack Provider v1alpha2 (v0.2)               |   |   | ✓ |   |   |   |   |   |   |
+|OpenStack Provider v1alpha3 (v0.3)               |   |   |   | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
```

Comment:
How do we now we support 1.21 with CAPO v0.3? (not sure what our tests on the relesae branch are testing)